vizro-core/src/vizro/static/css/card.css

```diff
@@ -23,9 +23,11 @@
 }
 
 .card_button {
-  height: 84%;
+  height: 100%;
+  left: 0;
   position: absolute;
-  width: 92%;
+  top: 0;
+  width: 100%;
```

Comment:
Small readability improvement suggestion (order of css properties):
```
.card_button {
  position: absolute;
  left: 0;
  top: 0;
  height: 100%;
  width: 100%;
}
```

Reply:
That's a good point and one we have also considered! There are many advantages of using that approach as well, but we opted for alphabetical order in the end :) 

See this article for reference: https://jerrylowm.medium.com/alphabetize-your-css-properties-for-crying-out-loud-780eb1852153

It's also natively supported by PyCharm, so it does the alphabetical ordering for you

Reply:
I just read the article and it seems to me now that it makes sense to keep them in alphabetical order. Thanks for the link. 😃